components/ContentItem.tsx

```diff
@@ -0,0 +1,209 @@
+      } else if (array.length > index + 1) {
+        return sum + c + ' & '
+      } else {
+        return sum + c
+      }
+    }, '')
+
+    return (
+      <div className="creators">
+        {creatorList}
+      </div>
+    )
+  }
+
+  const metadata = () => {
+    return (
+      <div className="metadata">
+        {item.version ? 'Version ' + item.version + ' of ' : ''}{item.types.resourceType ? startCase(item.types.resourceType) : 'Content'} published {item.publicationYear} via {item.publisher}
+      </div>
+    )
+  }
+
+  const description = () => {
+    if (!item.descriptions[0]) return ''
+
+    const descriptionHtml = truncate(item.descriptions[0].description, { 'length': 750, 'separator': '… '})
+
+    return (
+      <div className="description">
+        {ReactHtmlParser(descriptionHtml)}
+      </div>
+    )
+  }
+
+  const metricsCounter = () => {
+    if (item.citationCount + item.viewCount + item.downloadCount == 0) {
+      return (
+        <div className="metrics-counter">
+          <i><FontAwesomeIcon icon={faInfoCircle}/> No citations, views or downloads reported.</i>
+        </div>
+      )
+    }
+
+    return (
+      <div className="metrics-counter">
+        {item.citationCount > 0 &&
+          <i><FontAwesomeIcon icon={faQuoteLeft}/> <Pluralize singular={'Citation'} count={item.citationCount} /> </i>
+        }
+        {item.viewCount > 0 &&
+          <i><FontAwesomeIcon icon={faEye}/> <Pluralize singular={'View'} count={item.viewCount} /> </i>
+        }
+        {item.downloadCount > 0 &&
+          <i><FontAwesomeIcon icon={faDownload}/> <Pluralize singular={'Download'} count={item.downloadCount} /> </i>
+        }
+      </div>
+    )  
+  }
+
+  const bookmark = (
+    <Popover id="bookmark" title="Bookmarking">
+      Bookmarking on this site will be implemented later in 2020. <a href="https://datacite.org/roadmap.html" target="_blank" rel="noreferrer">Provide input</a>
+    </Popover>
+  )
+
+  const share = (
+    <Popover id="share" title="Sharing via Social Media">
+      Sharing via social media will be implemented by August 2020. <a href="https://datacite.org/roadmap.html" target="_blank" rel="noreferrer">Provide input</a>
+    </Popover>
+  )
+
+  const save = (
+    <Popover id="save" title="Download Metadata">
+      Downloading metadata will be implemented by August 2020. <a href="https://datacite.org/roadmap.html" target="_blank" rel="noreferrer">Provide input</a>
+    </Popover>
+  )
+
+  const cite = (
+    <Popover id="cite" title="Formatted Citation">
+      Formatted citations will be implemented by August 2020. <a href="https://datacite.org/roadmap.html" target="_blank" rel="noreferrer">Provide input</a>
+    </Popover>
+  )
+
+  const claim = (
+    <Popover id="claim" title="Claim to ORCID Record">
+      Claiming to an ORCID record will be implemented later in 2020. <a href="https://datacite.org/roadmap.html" target="_blank" rel="noreferrer">Provide input</a>
```

Comment:
Wouldn't be better to write something more vague (i.e., avoid dates) w/r/t when these features would be included ? Not even **public roadmap** goes into this level of detail.

Reply:
This comes from the FREYA description of work, where we said we will do formatted citations and ORCID claiming by October 2020.

The only exception is bookmarks, which we haven't discussed yet. But that is also because we haven't discussed much what end users should be able to do.